packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -668,118 +392,6 @@ Future<void> main() async {
       expect(currentUrl, secondaryUrl);
     });
   });
-
-  testWidgets('target _blank opens in same window',
-      (WidgetTester tester) async {
-    final Completer<void> pageLoaded = Completer<void>();
-
-    final WebViewController controller = WebViewController()
-      ..setJavaScriptMode(JavaScriptMode.unrestricted)
-      ..setNavigationDelegate(NavigationDelegate(
-        onPageFinished: (_) => pageLoaded.complete(),
-      ));
-
-    await tester.pumpWidget(WebViewWidget(controller: controller));
-
-    await controller.runJavaScript('window.open("$primaryUrl", "_blank")');
-    await pageLoaded.future;
-    final String? currentUrl = await controller.currentUrl();
-    expect(currentUrl, primaryUrl);
-  });
-
-  testWidgets(
-    'can open new window and go back',
-    (WidgetTester tester) async {
-      Completer<void> pageLoaded = Completer<void>();
-
-      final WebViewController controller = WebViewController()
-        ..setJavaScriptMode(JavaScriptMode.unrestricted)
-        ..setNavigationDelegate(NavigationDelegate(
-          onPageFinished: (_) => pageLoaded.complete(),
-        ))
-        ..loadRequest(Uri.parse(primaryUrl));
-
-      await tester.pumpWidget(WebViewWidget(controller: controller));
-
-      expect(controller.currentUrl(), completion(primaryUrl));
-      await pageLoaded.future;
-      pageLoaded = Completer<void>();
-
-      await controller.runJavaScript('window.open("$secondaryUrl")');
```

Comment:
Do you know how this works on other platforms?

Reply:
You mean `window.open` doesn't actually open a new window on Android?

Reply:
Yes, I know that if I do not specify a target, it will be output to the same Window.
In ewk, when I tested it, Window.open () returned NULL.
I couldn't find to the exact specifications of the ewk,so I think it as an unsupported.